frequently-encountered-problems.html

```diff
@@ -200,6 +200,39 @@ <h2>Using NTLM authentication with Node.js 17</h2>
         </p>
     </dd>
     <br />
+    <dt>
+        <h2>The relations between <code>options.trustServerCertificate, options.encrypt,</code> and <code>options.cryptoCredentialDetails</code></h2>
+    </dt>
+    <dd>
+        <ul>
+            <li>
+                If <code>encrypt = true</code> and <code>trustServerCertificate = false</code>, then <code>cryptoCredentialsDetails</code> must be provided. 
+                Details on how to provide the <code>cryptoCredentialsDetails</code> object can be found in the <a href="/api-connection.html">connection</a> page of the documentation.
+            </li>
+            <li>
+                If <code>encrypt = false</code>, then <code>trustServerCertificate</code> is ignored.
+            </li>
+        </ul>
+
+    </dd>
+    <br />
+    <dt>
+        <h2>TLS 1.0 compatibility with Node 12 and newer</code></h2>
+    </dt>
+    <dd>
+        <p>
+            The default TLS settings were tightened in Node 12 and newer, requiring TLS 1.2 as the minimum version. You can find documentation
+            on how to upgrade your server to TLS 1.2 
```

Comment:
just a reminder in case we forget: we can say 1.2 or above, just to be safe.

Reply:
Good point. Added it